### PR TITLE
[3065] Update accessibility statement after audit

### PR DESCRIPTION
**Pull request by aliuk2012**
### Context
Update accessibility statement after an audit

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product review

### CODE REVIEW

**Comment by aliuk2012** [comment]
[Update accessibility statement](https://trello.com/c/vtCs7TOt/3065-update-accessibility-statement)

File: app/views/pages/accessibility.html.erb [hunk 1]
```diff
@@ -3,37 +3,77 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">Accessibility statement for Find teacher training courses</h1>
```
Comment (defong) [inline app/views/pages/accessibility.html.erb]: Is that not meant to be `Find postgraduate teacher training`?

File: app/views/pages/accessibility.html.erb [hunk 2]
```diff
@@ -3,37 +3,77 @@
...
 
       <h3 class="govuk-heading-m">Reporting accessibility problems with Find postgraduate teacher training</h3>
 
-      <p class="govuk-body">If you think we’re not meeting accessibility requirements, or you’d like to access any information in a different format (such as accessible PDF, large print, easy read, audio recording or braille), contact <%= mail_to "becomingateacher@digital.education.gov.uk", "becomingateacher@digital.education.gov.uk", class: "govuk-link" %>. Include 'Accessibility issues' in the subject line of your email.</p>
+      <p class="govuk-body">
+        If you think we’re not meeting accessibility requirements, or you’d like to access
+        any information in a different format (such as accessible PDF, large print, easy read,
+        audio recording or braille), contact
+        <%= mail_to "becomingateacher@digital.education.gov.uk", "becomingateacher@digital.education.gov.uk", class: "govuk-link" %>.
+        Include 'Accessibility issues' in the subject line of your email.
```
Comment (defong) [inline app/views/pages/accessibility.html.erb]: should we do the subject line `Accessibility issues` as well in the mailto?

**Comment by aliuk2012** [comment]
I will squash the commits once approved. Paul G is reviewing the preview app.

File: app/views/pages/accessibility.html.erb (hunk 2 above)
Comment (aliuk2012) [reply app/views/pages/accessibility.html.erb]: That's a good idea, We might want to still include the copy but I'll check with Paul G

Comment (aliuk2012) [reply app/views/pages/accessibility.html.erb]: We will remove this line and add the copy to the subject line for the user

Comment (aliuk2012) [reply app/views/pages/accessibility.html.erb]: Even better, we should use `bat_contact_mail_to` helper method to generate this link.

**Pull request merged**